zeus/device/cpu/rapl.py

```diff
@@ -90,31 +90,23 @@ def __str__(self) -> str:
         return f"Path: {self.path}\nEnergy_uj_path: {self.energy_uj_path}\nName: {self.name}\
         \nLast_energy: {self.last_energy}\nMax_energy: {self.max_energy_range_uj}"
 
-    def _read(self) -> int:
+    def read(self) -> float:
         """Read the current energy value from the energy_uj file.
 
         Returns:
             int: The current energy value in millijoules.
```

Comment:
```suggestion
        Returns:
            The current energy value in millijoules.
```
Refer to the Google style guide.